src/main/java/Sinep/Event.java

```diff
@@ -0,0 +1,29 @@
+package Sinep;
+public class Event extends Task {
+    protected String startDateTime;
+    protected String endDateTime;
+    public Event(String description) {
+        super(description);
+        try {
+            String[] givenTimeline = description.split("/from ", 2);
```

Comment:
It is advisable to have logically linked code to be put together, and other part separated by a line break.

Reference: https://nus-cs2113-ay2324s2.github.io/website/se-book-adapted/chapters/codeQuality.html#structure-code-logically

```suggestion
    protected String startDateTime;
    protected String endDateTime;
    
    public Event(String description) {
        super(description);
        
        try {
            String[] givenTimeline = description.split("/from ", 2);
```